WORKSPACE.bazel

```diff
@@ -17,17 +17,6 @@ workspace(name = "stablehlo")
 
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
-SKYLIB_VERSION = "1.3.0"
-
-http_archive(
```

Comment:
Will the other `http_archive`s in this file also need to be migrated?

Reply:
Eventually -- but LLVM needs to change to support bazelmod.

I spoke to Bazel folks on Slack and they said for now no need to port.

https://bazelbuild.slack.com/archives/CA31HN1T3/p1707853906929909?thread_ts=1707845158.715749&cid=CA31HN1T3

Reply:
Cool, thanks for looking into it!